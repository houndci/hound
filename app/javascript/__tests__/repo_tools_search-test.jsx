import RepoToolsSearch from '../components/ReposContainer/components/RepoTools/RepoToolsSearch';

it('renders appropriately', () => {
  const onSearchInput = jest.fn();

  const wrapper = shallow(
    <RepoToolsSearch onSearchInput={onSearchInput} />
  );
  expect(wrapper).toMatchSnapshot();
});
