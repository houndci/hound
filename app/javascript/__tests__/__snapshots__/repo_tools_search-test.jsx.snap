exports[`test renders appropriately 1`] = `
<div
  className="repo-tools-search">
  <input
    className="repo-search-tools-input"
    onChange={[Function]}
    type="text" />
</div>
`;
